tests/foreman/ui/test_contentview.py

```diff
@@ -2297,3 +2297,157 @@ def test_positive_publish_promote_with_rh_ostree_and_other(self):
+                             "view_search",
+                             "view_cases",
+                             "view_log_viewer",
+                             "view_activation_keys",
+                             "create_activation_keys",
+                             "edit_activation_keys",
+                             "destroy_activation_keys",
+                             "view_reports",
+                             "view_host_collections",
+                             "create_host_collections",
+                             "edit_host_collections",
+                             "destroy_host_collections",
+                             "view_usergroups",
+                             "view_subscriptions",
+                             "attach_subscriptions",
+                             "unattach_subscriptions",
+                             "import_manifest",
+                             "delete_manifest",
+                             "view_organizations",
+                             "view_content_views",
+                             "create_content_views",
+                             "edit_content_views",
+                             "destroy_content_views",
+                             "publish_content_views",
+                             "promote_or_remove_content_views",
+                             "view_sync_plans",
+                             "create_sync_plans",
+                             "edit_sync_plans",
+                             "destroy_sync_plans",
+                             "view_lifecycle_environments",
+                             "create_lifecycle_environments",
+                             "edit_lifecycle_environments",
+                             "destroy_lifecycle_environments",
+                             "promote_or_remove_content_views_to_environments",
+                             "view_products",
+                             "create_products",
+                             "edit_products",
+                             "destroy_products",
+                             "sync_products",
+                             "view_filters",
+                             "create_filters",
+                             "edit_filters",
+                             "destroy_filters"
+                             ]
+        for perm in custom_perms_list:
+            permissions = entities.Permission(name=perm).search()
+            entities.Filter(permission=permissions, role=roles).create()
+        username = gen_string('alpha')
+        user_password = gen_string('alphanumeric')
+        entities.User(
+                login=username,
+                role=[roles],
+                password=user_password,
+                organization=[self.organization],
+                ).create()
+        repo_name = gen_string('alpha')
+        env_name = gen_string('alpha')
+        cv_name = gen_string('alpha')
+        strategy, value = locators['content_env.select_name']
```

Comment:
You dont need this anymore, now you can simply do

```python
locators['content_env.select_name'] % "string"
```

BTW: I cant see `strategy or `value` being used in the code below, where do you used this?